lib/notifications/client.rb

```diff
@@ -45,7 +45,7 @@ def send_sms(args)
     # @return [Notification]
     def get_notification(id)
       Notification.new(
-        speaker.send(:get, id)["notification"]
+        speaker.send(:get, id)["data"]["notification"]
       )
     end
```

Comment:
Have made get/post public - Not sure who/why they where private methods in the first place.

With the interface I think you are right, Client is the wrong place for this knowledge. Let me know what you think of the change I proposed.